docs/Reference/CLI/Subcommands/Slashing-Protection.md

```diff
@@ -146,7 +146,186 @@ The file to export the slashing protection database to.
 
 Exports the database in the [validator client interchange format] format.
 
+## `repair`
+
+Repairs corrupted slashing-protection data files used by Teku.
+
+### check-only-enabled
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection repair --checking-only-enabled[=<BOOLEAN>]
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection repair --checking-only-enabled=false
+    ```
+
+Reads and reports potential slashing-protection file problems, but doesn't update any files.
+
+### config-file
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection repair --config-file=<FILE>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection repair --config-file=/home/me/me_node/config.yaml
+    ```
+
+The path to the YAML configuration file.
+The default is `none`.
+
+### data-base-path, data-path
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection repair --data-base-path=<PATH>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection repair --data-base-path=/home/me/me_node
+    ```
+
+The path to the Teku data directory. The default directory is OS dependent:
+
+* macOS: `~/Library/teku`
+* Unix/Linux: `$XDG_DATA_HOME/teku` if `$XDG_DATA_HOME` is set; otherwise `~/.local/share/teku`
+* Windows: `%localappdata%\teku`.
+
+The default Docker image location is `/root/.local/share/teku`.
+
+### data-validator-path
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection repair --data-validator-path=<PATH>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection repair --data-validator-path=/home/me/me_validator
+    ```
+
+Path to validator client data. Defaults to `<data-base-path>/validator` where `<data-base-path>`
+is specified using [`--data-base-path`](#data-base-path-data-path).
+
+### eth1-deposit-contract-address
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection repair --eth1-deposit-contract-address=<ADDRESS>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection repair --eth1-deposit-contract-address=0x77f7bED277449F51505a4C54550B074030d989bC
+    ```
+
+Ethereum 1.0 address of the deposit contract. Only required when creating a custom network.
+
+### initial-state
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection repair --initial-state=<STRING>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection repair --initial-state=/home/me/genesis.ssz
+    ```
+
+Path or URL to an SSZ-encoded state file. The state file can be used to specify the genesis state,
+or a [recent finalized checkpoint state from which to sync].
+
+This option does not need to be specified if the initial state is provided by the network specified
+using the [`--network`](#network) option.
```

Comment:
these options won't help a slashing protection repair, i'd probably just suggest not listing in documentation.

Reply:
I realised --network may actually be useful when i was answering above - in the circumstance where you're not specifying your config-file...